tests/integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -167,6 +167,24 @@ if [ -f "${arch_config_file}" ]; then
 fi
 
 if policy_tests_enabled; then
```

Comment:
The issue occurs exclusively on the TEE platforms, and `policy_tests_enabled` function serves to check the platforms. I think we should implement a new function to tackle the issue, rather than relying on the existing one.
```bash
policy_tests_enabled() {
	# The Guest images for these platforms have been built using AGENT_POLICY=yes -
	# see kata-deploy-binaries.sh.
	[ "${KATA_HYPERVISOR}" == "qemu-sev" ] || [ "${KATA_HYPERVISOR}" == "qemu-snp" ] || \
		[ "${KATA_HYPERVISOR}" == "qemu-tdx" ] || [ "${KATA_HOST_OS}" == "cbl-mariner" ]
}
```

Reply:
But now I move the code to `reset_k8s_images` and call it in `deploy_nydus_snapshotter`.